tests/e2e/operator.sh

```diff
@@ -132,29 +132,32 @@ install_ccruntime() {
 	kubectl create -k .
 	popd >/dev/null
 
-	local pod=""
-	local cmd=""
-	for pod in cc-operator-daemon-install cc-operator-pre-install-daemon; do
-		cmd="kubectl get pods -n '$op_ns' --no-headers |"
-		cmd+="egrep -q ${pod}.*'\<Running\>'"
-		if ! wait_for_process 600 30 "$cmd"; then
-			echo "::error:: $pod pod is not running"
-			local pod_id
-			pod_id="$(get_pods_regex "$pod" "$op_ns")"
-			echo "::debug:: Pod $pod_id"
-			debug_pod "$pod_id" "$op_ns"
-
-			return 1
+	# Wait for the operator to report installationStatus.completed on all nodes
+    local count=0
+	local nodes
+
+    while true; do
+        nodes="$(kubectl get ccruntime -o jsonpath='{range .items[*]}{.status.totalNodesCount}{end}')" || nodes=-1
```

Comment:
Very good idea getting info from the `ccruntime` resource...I will use that code in the future I'm sure :)